src/iTunesLibrary/ITCompat.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+
+namespace iTunesLibrary
+{
+#if !XAMCORE_2_0

```

Comment:
close :) I should have mentioned it but `XAMCORE_2_0` is a few years back (when we did 32/64 unified)

What you need if `#if XAMCORE_4_0` so, the next time we read API compatibility (if ever), that code can be removed.

and, just to paint a complete picture, `XAMCORE_3_0` was used to exclude stuff when we added tvOS and watchOS support (so we did not bring `[Obsolete]`d API in the new platforms.

Reply:
Thanks a lot for the detailed explanations, I'm learning so much about the project everyday !